Help/FAQ.rst

```diff
@@ -55,6 +55,15 @@ There are also generic bears, which can be applied language independently on
 your code. Their capabilities and information can be seen
 `here <https://coala.io/#/languages>`__.
 
+What is a bear anyway?
+----------------------
+
+A bear is a set of rules that is used by coala packages to check for
+lint issues and fix them if found. A bear is used for defining certain
+quality requirements in various languages. While many of these
+plugins(bears) for various languages are shipped along with coala,
+you can find various others on the internet.
```

Comment:
My suggestion would be to change
- `find various others on the internet` => `easily write your own`